app/components/ui/atoms/SelectedItem.js

```diff
@@ -2,16 +2,16 @@ import React from 'react'
 import styled from 'styled-components'
 import { th } from '@pubsweet/ui-toolkit'
 
-import Icon from './Icon'
+import Icon from '../atoms/Icon'
+import ButttonAsIconWraper from './ButtonAsIconWrapper'
 
-const CrossIcon = props => (
+const CrossIcon = ({ overrideName, ...props }) => (
   <Icon iconName="X" overrideName="@pubsweet-pending.Tags.Remove" {...props} />
 )
 
-const StyledIcon = styled(CrossIcon)`
-  margin-left: ${th('space.1')};
-  cursor: pointer;
+const StyledRemoveButton = styled(ButttonAsIconWraper)`
   fill: ${th('colorTextReverse')};
+  margin-left: ${th('space.1')};
```

Comment:
Yeah that's a good point about the naming, if nothing else.

Me and @dryhten had a bit of a back and forth between 3 potential implementations of `ButtonAsIconWrapper`, which were along these lines too. Should that component be purely responsible for styling a native <button> element, or actually be a React element that knows it will have a child passed to it?

Similar thing here. Is it simpler to have the structure defined in the render function & rename this as `StyledButtonWrapper`, or essentially give it a child within the styling block.

:thinking: :face_with_head_bandage:

Reply:
🤷‍♂️  happy for this to go in as-is – up to you.